hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+:obj:`hypothesis.stateful.GenericStateMachine` and
+:obj:`hypothesis.stateful.RuleBasedStateMachine` now raise an explicit error
+when instances of :obj:`hypothesis.settings` are assigned to the classes'
+settings attribute. Error directs users to use correct assignment methods.
```

Comment:
Instead of saying that the error directs users to correct assignment methods, we should just do that here too.  e.g.

> ...classes' settings attribute, which is a no-op (:issue:\`1643\`).  Instead, users should assign to \`\`SomeStateMachine.TestCase.settings\`\`, or use \`\`@settings(...)\`\` as a class decorator to handle this automatically.